### ISSUE TITLE
Parameter 'output_activation' ignored when 'deep_supervision' is False.

### ISSUE THREAD

**Issue by thupalo** [yingkaisha/keras-unet-collection#3]
Hi,
First of all - thank you for this great model collection.
Maybe this is intended by design but:
`base_model = models.unet_3plus_2d(input_shape, n_labels=19, filter_num_down=[64, 128, 256, 512, 1026], 
                             filter_num_skip=[64, 64, 64, 64], filter_num_aggregate=256, 
                             stack_num_down=2, stack_num_up=1, activation='ReLU', output_activation='Sigmoid',
                             batch_norm=True, pool='max', unpool=False, deep_supervision=False, name='unet3plus')`
then the call
`base_model.layers[-1].__class__`
returns
`tensorflow.python.keras.layers.advanced_activations.ReLU`
I'd rather expected a 'sigmoid' activation.

**Comment by yingkaisha** [comment]
Hi,

The problem is identified, I will push a fix shortly.

Thank you.

**Comment by yingkaisha** [comment]
`pip install keras-unet-collection==0.0.17` see if the problem is solved.

**Comment by thupalo** [comment]
Looks great. Thank you.
Stay safe.
